cmd/file-mod/commands/root.go

```diff
@@ -3,6 +3,7 @@ package commands
 import (
 	"os"
 
+	"github.com/prometheus/common/log"
```

Comment:
Same here, I think this is my fault because I put `log.Error` in a suggestion, but forgot that we typically are using logrus which has an `Error` method.  If you want to use the stdlib logger then you'll want `log.Print` or one of the other methods here: https://golang.org/pkg/log/